src/preact/context.js

```diff
@@ -15,7 +15,7 @@
  */
 
 import * as Preact from './index';
-import {Loading, reducer as loadingReducer} from '../loading';
+import {Loading, reducer as loadingReducer} from './loading-instructions';
```

Comment:
This removes the only restricted dependency, so this file was removed from the allowlist